src/main/java/net/unicon/lti/utils/DomainUtils.java

```diff
@@ -0,0 +1,43 @@
+package net.unicon.lti.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class DomainUtils {
+
+    public static String insertDomain(String altDomain, String placement) {
+
+        StringBuffer buf = new StringBuffer(placement);
+        int dot = placement.indexOf(".");
+        buf.insert(dot,  "-" + altDomain);
+        return buf.toString();
+    }
+
+    public static String extractDomain(String host){
+
+        StringBuffer buf = new StringBuffer(host);
+        int dot = host.indexOf(".");
+        String toFindMinus = buf.substring(0, dot);
+        int start = toFindMinus.lastIndexOf("-");
+        if (start == -1){
+            return null;
+        }
+        String prefix = buf.substring(start+1, dot);
+
+        Set<String> defaultPrefixes = new HashSet<>();
```

Comment:
Should this set go in the TextConstants.java file?

Reply:
I can move it there.

Reply:
Moved :)